config/makefile

```diff
@@ -110,4 +110,5 @@ config-mk:
 
 clean:
 	rm -f $(CONFIG_HPP) $(CONFIG_MK) sample-runs-build.log
-	rm -f $(GHV) $(GHV).out $(GMV) $(GMV).out *.dSYM
+	rm -f $(GHV) $(GHV).out $(GMV) $(GMV).output
```

Comment:
```suggestion
	rm -f $(GHV) $(GHV).out $(GMV) $(GMV).out
```

Reply:
reverted in https://github.com/mfem/mfem/pull/4716/commits/b1537594402eaa0c9dea2b3fc8e1640cf0131a96